html5lib/html5parser.py

```diff
@@ -1706,6 +1706,7 @@ def startTagColgroup(self, token):
 
         def startTagCol(self, token):
             self.startTagColgroup(impliedTagToken("colgroup", "StartTag"))
+            token["selfClosingAcknowledged"] = True
```

Comment:
Oh, it doesn't. Hopefully that's just changing `token` to `new_token` in the loop.